common/types/types.go

```diff
@@ -10,4 +10,5 @@ type MeshedSuperNode struct {
 type NftRegMetadata struct {
 	CreatorPastelID string
 	BlockHash       string
+	BurnTxID        string
```

Comment:
I think should define another structure for action registration, like `ActionRegMetadata`?

Reply:
done, create a new type for actionreg.